site/_posts/2021-08-18-annnoucing-fusionauth-1.30.md

```diff
@@ -0,0 +1,89 @@
+ - Failed login
+ - Forgot password
+ - Send email verification
+ - Send passwordless
+ - Send registration verification
+ - Send two-factor
+ 
+Capcha - Uses additonal (re)captcha to guard against brute force attacks. 
+- Adds a (re)captcha for login pages which use an email/password combination
+- Adds support for Google ReCaptcha v2, Google ReCaptcha v3, HCaptcha and Hcaptcha Enterprise
+
+IP Location Services
+- Includes the IP address of the geographic location where a Forgot/Reset Password email was created, so the recipient can identify if the location is genuinely their request or not. 
+- Flags suspicious IP adddresses on login and sends an email to notify the user of a new login with the IP's approximate location
+- Calculates 'impossible travel' to see if a user could genuinely login in different locations around the globe in a reasonable time frame. (This does not provide support if your user is a quantum being...)
+- With unexpected IP addresses, sends a user an email to notify them of a new login with an approximate location of the IP address. 
+
+To learn more, you can take a look at our [release notes](/docs/v1/tech/release-notes/#version-1-30-0). 
+
+## Webhook Events
+
+We've added 38 webhook events to help you moniter what is going on with your app.  These webhooks cover an entire breadth of createing, updated, and deleting a user, as well as the user registration and password reset lifecycle. Lastly, we've added some webhook events to moniter any kickstart files you may create, as well as moniter JWT behavior with refresh tokens and public/private keypairs are updated. 
+
+## The rest of it
+
+Some of the other enhancements and fixes included in this release:
+
+* search on `oldValue`, `newValue`, and `reason` in the Audit Log
+* More themed pages have access to the currently logged in user using the `currentUser` varible
+* Using an IdP with a linking mode other than Create Pending Link, the token is now stored correctly in the Identity Provider Link
+* Making an API request to `/api/two-factor/login` with an empty JSON body, an exception now occurs instead of returning a validation error with a `400` status code. 
+
+## Upgrade at will
+
+The [release notes](/docs/v1/tech/release-notes/#version-1-30-0) are a guide of the changes, fixes, and new features. Please read them carefully to see if any features you use have been modified.
+
+If you'd like to upgrade your self-hosted FusionAuth instance, see our [upgrade guide](/docs/v1/tech/installation-guide/upgrade/). 
+
+If you have a FusionAuth Cloud deployment, proceed to the "Deployments" tab on your [account dashboard](https://account.fusionauth.io/account/deployment/){:target="_blank"} and upgrade your servers. If you have any questions about the upgrade, [please open a support ticket](https://account.fusionauth.io/account/support/){:target="_blank"}.
+
+Or, if we've piqued your interest and you'd like to use FusionAuth, [check out your options](/pricing/).
```

Comment:
I am not sure why this link looks strange to me 
[check out your options](/pricing/).

could be fine...